modules/movies.js

```diff
@@ -1,33 +1,94 @@
+// -------------------- STARTER CODE -------------------- //
+
+'use strict';
+
 const axios = require('axios');
+const cache = require('./cache.js');
 
 async function movies(req, res) {
   const movieQuery = req.query.searchQuery;
-  console.log('MOVIE QUERY???', movieQuery);
-
-  try {
-    const moviesKey = process.env.MOVIE_API_KEY;
-    const movies_API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${moviesKey}&query=${movieQuery}&language=en-US`;
-
-    const moviesAPIRes = await axios.get(movies_API_URL);
-    console.log('MOOOOVIIIEEEE DAAATTAAAAA', moviesAPIRes.data.results);
-    const movies = moviesAPIRes.data.results.map(obj => new Films(obj));
-    console.log('LOOK AT ME!!', movies);
-    res.send(movies);
-  } catch (error) {
-    res.status(500).send('No movies found.');
+  const moviesKey = process.env.MOVIE_API_KEY;
+  const movies_API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${moviesKey}&query=${movieQuery}&language=en-US`;
+
+  if (cache[moviesQuery] && (Date.now() - cache[moviesQuery].timestamp < 36500)) {
+    console.log('Cache hit');
+    res.send(cache[moviesQuery]);
```

Comment:
Hi Ryan! 
  Nice work so far here!

Take a look at this line here, slap a console.log in and see what it shows :) Are you accessing the data?